book/actix-plugin.md

```diff
@@ -9,48 +9,21 @@ Let's start with a simple actix-web application. It has `actix-web` and `serde`
 
 [dependencies]
 # actix-web 2.0 is supported through "actix2" and "actix2-nightly" features
-actix-web = "3.0"
+# actix-web 3.0 is supported through "actix3" and "actix3-nightly" features
+actix-web = "4.0"
 # The "actix-nightly" feature can be specified if you're using nightly compiler. Even though
 # this plugin works smoothly with the nightly compiler, it also works in stable
 # channel (replace "actix-nightly" feature with "actix" in that case). There maybe compilation errors,
 # but those can be fixed.
 # Add the "v3" option if you want to expose an OpenAPI v3 document
 paperclip = { version = "0.6", features = ["actix"] }
-serde = "1.0"
+serde = { version = "1.0", features = ["derive"] }
 ```
 
 Our `main.rs` looks like this:
 
 ```rust
-use actix_web::{App, HttpServer, web::{self, Json}};
-use serde::{Serialize, Deserialize};
-
-#[derive(Serialize, Deserialize)]
-struct Pet {
-    name: String,
-    id: Option<i64>,
-}
-
-async fn echo_pet(body: Json<Pet>) -> Result<Json<Pet>, ()> {
-    Ok(body)
-}
-
-#[actix_web::main]
-async fn main() -> std::io::Result<()> {
-    HttpServer::new(|| App::new()
-        .service(
-            web::resource("/pets")
-                .route(web::post().to(echo_pet))
-        )
-    ).bind("127.0.0.1:8080")?
-    .run().await
-}
-```
-
-Now, let's modify it to use the plugin!
-
-```rust
-use actix_web::{App, HttpServer};
+use actix_web::{App, HttpServer, Error};
```

Comment:
Any reason to remove the rest of the code? (sorry If I'm misinterpreting)

Reply:
You're right, I somehow deleted the pre-actix example. I'll amend